[2024-02-21T07:38:08.239+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-08T00:00:00+00:00 [queued]>
[2024-02-21T07:38:08.259+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-08T00:00:00+00:00 [queued]>
[2024-02-21T07:38:08.259+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:38:08.277+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-08 00:00:00+00:00
[2024-02-21T07:38:08.285+0000] {standard_task_runner.py:60} INFO - Started process 114 to run task
[2024-02-21T07:38:08.287+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-08T00:00:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpmdjdxdnb']
[2024-02-21T07:38:08.289+0000] {standard_task_runner.py:88} INFO - Job 80: Subtask get_mail
[2024-02-21T07:38:08.334+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-08T00:00:00+00:00 [running]> on host 893865381b0d
[2024-02-21T07:38:08.416+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-08T00:00:00+00:00'
[2024-02-21T07:38:08.416+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 196, in execute
    context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 261, in context_merge
    context.update(*args, **kwargs)
  File "/usr/local/lib/python3.8/_collections_abc.py", line 837, in update
    for key, value in other:
ValueError: too many values to unpack (expected 2)
[2024-02-21T07:38:08.422+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240208T000000, start_date=20240221T073808, end_date=20240221T073808
[2024-02-21T07:38:08.430+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 80 for task get_mail (too many values to unpack (expected 2); 114)
[2024-02-21T07:38:08.464+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T07:38:08.483+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T07:59:47.878+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-08T00:00:00+00:00 [queued]>
[2024-02-21T07:59:47.899+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-08T00:00:00+00:00 [queued]>
[2024-02-21T07:59:47.900+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:59:47.926+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-08 00:00:00+00:00
[2024-02-21T07:59:47.936+0000] {standard_task_runner.py:60} INFO - Started process 382 to run task
[2024-02-21T07:59:47.940+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-08T00:00:00+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmp12ko0qyq']
[2024-02-21T07:59:47.942+0000] {standard_task_runner.py:88} INFO - Job 132: Subtask get_mail
[2024-02-21T07:59:48.012+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-08T00:00:00+00:00 [running]> on host 9bab6a0db245
[2024-02-21T07:59:48.169+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-08T00:00:00+00:00'
[2024-02-21T07:59:50.631+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T07:59:53.628+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T07:59:53.634+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-194.txt')
[2024-02-21T07:59:53.635+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.636+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T07:59:53.641+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-195.txt')
[2024-02-21T07:59:53.642+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.643+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T07:59:53.646+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-196.txt')
[2024-02-21T07:59:53.646+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.647+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T07:59:53.649+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-197.txt')
[2024-02-21T07:59:53.649+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.650+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T07:59:53.652+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-198.txt')
[2024-02-21T07:59:53.652+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.653+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T07:59:53.654+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-199.txt')
[2024-02-21T07:59:53.655+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.655+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T07:59:53.657+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-200.txt')
[2024-02-21T07:59:53.657+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.657+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T07:59:53.659+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-201.txt')
[2024-02-21T07:59:53.659+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.659+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T07:59:53.661+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-202.txt')
[2024-02-21T07:59:53.661+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.661+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T07:59:53.662+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-203.txt')
[2024-02-21T07:59:53.663+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.245+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T07:59:54.254+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240208T000000, start_date=20240221T075947, end_date=20240221T075954
[2024-02-21T07:59:54.300+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T07:59:54.330+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
